autoload/ctrlp.vim

```diff
@@ -1789,7 +1789,10 @@ endf
 
 fu! ctrlp#setdir(path, ...)
 	let cmd = a:0 ? a:1 : 'lc!'
-	sil! exe cmd s:fnesc(a:path, 'c')
+	let newcwd = s:fnesc(a:path, 'c')
+	if getcwd(0) != newcwd
```

Comment:
Why 0?

Reply:
same as `getcwd()`, guess I could have just used that. Anyway I have since found other ways around my issue, so I don't need this anymore and I don't think anyone else is interested in this change so I'll just close it